pkg/solana/transmitter.go

```diff
@@ -81,18 +80,18 @@ func (c *ContractTracker) Transmit(
 
 	// Send transaction, and wait for confirmation:
 	go func() {
-		txSig, err := confirm.SendAndConfirmTransactionWithOpts(
+		txSig, err := c.client.rpc.SendTransactionWithOpts(
 			context.Background(), // does not use libocr transmit context
-			c.client.rpc,
-			c.client.ws,
 			tx,
-			true, // skip preflight
+			false, // use preflight as first check
 			rpc.CommitmentConfirmed,
 		)
 
 		if err != nil {
 			c.lggr.Errorf("error on Transmit.SendAndConfirmTransaction: %s", err.Error())
 		}
+		// TODO: poll rpc for tx confirmation (WS connection unreliable)
```

Comment:
Was there an agreement we are dropping WS support for the long term?

I'd prefer we actually have two implementations, both simpler but more resource-intensive RPC polling, and WS subscription so we can make incremental improvements to it and potentially switch again once stable enough.

Reply:
Sort of, there was consensus for dropping it for now. 

I agree that the RPC polling is more resource intensive, but the WS implementation in the solana-go module was the cause of a memory leak which would take down the core node. Building a stable WS implementation would take a lot of time and effort, so it was shelved for now.

Reply:
I'd just suggest we declare a TxConfirmer as an interface and then we implement the RPC TxConfirmer + some docs about what were the latest encountered issues using the WS TxConfirmer (no need to do any work on it now). That way we have a defined path of switching implementations if/when WS implementation becomes stable.

Reply:
that sounds like a good idea, but probably better addressed in a separate PR?
it will also likely tie into some tx management work for terra as well

Reply:
Yup, no need to slow down this PR whose main purpose is to quickly fix the memory leak and get the network stable.

I'll open a GH Issue to track this suggestion for the future. And yeah, we should also define types like `TxCofirmer` on the Relay SDK level and use (implement) them for specific relays.

Reply:
might be easier to add comments to this open issue for adding a tx confirmer: https://github.com/smartcontractkit/chainlink-solana/issues/90